src/main/java/jp/ikedam/jenkins/plugins/updatesitesmanager/ManagedUpdateSite.java

```diff
@@ -235,6 +319,58 @@ public FormValidation doCheckCaCertificate(
             
             return FormValidation.ok();
         }
-        
+
+        public FormValidation doCheckCredentialsId(@CheckForNull @AncestorInPath Item item,
+                                                   @QueryParameter String credentialsId,
+                                                   @QueryParameter String serverUrl) {
+            if (item == null) {

```

Comment:
that procedure comes from documentation that explain a secure way how to implement the method (it is also present in some other plugins).
It could be null in case call is done manually. Since is signature is marked  `@ChecForNull` this check must be performed.